### PR TITLE
Release: 1.6.1

### PR DESCRIPTION
**Pull request by aboedo**
- Adds missing availability check for `simulatesAskToBuyInSandbox`
    https://github.com/RevenueCat/purchases-hybrid-common/pull/66

### CODE REVIEW

**Comment by aboedo** [comment]
caught by cocoapods, I forgot to add this check 🤦

File: RELEASING.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 - Run `fastlane bump_and_update_changelog version:X.Y.Z` (where X.Y.Z is the new version) to update the version number in `android/build.gradle`, `android/gradle.properties` and `PurchasesHybridCommon.podspec`
 - Update purchases-android version in `android/build.gradle`
 - Update purchases-ios pod version in `PurchasesHybridCommon.podspec` and `ios/PurchasesHybridCommon/Podfile`.
+- run `pod lib lint` to ensure that no problems are found with the pod.
```
Comment (vegaro) [inline RELEASING.md:8]: Let's make sure we add this as part of CI at some point

**Review by vegaro — APPROVED** [review]

File: RELEASING.md (hunk 1 above)
Comment (aboedo) [reply RELEASING.md:8]: +100

Comment (aboedo) [reply RELEASING.md:8]: added to backlog

**Pull request merged**